lib/api/objectPut.js

```diff
@@ -17,7 +17,7 @@ function _storeInMDandDeleteData(bucketName, dataGetInfo, cipherBundle,
             if (err) {
                 return callback(err);
             }
-            if (dataToDelete) {
+            if (dataToDelete && dataToDelete[0].dataStoreName !== 'kinetic') {
```

Comment:
Please add in this commit that the kinetic backend does not handle the batch delete yet.